.github/mergify.yml

```diff
@@ -16,9 +15,9 @@ pull_request_rules:
     conditions:
       - author=snyk-bot
       - check-success~=lint
-      - check-success~=build
```

Comment:
may i get context for this regarding why this line was removed?

Reply:
added back in, removed by accident.